app/views/shared/forms/_percentage_home_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with(model: @form,
+              scope: :legal_aid_application,
+              url: url,
+              method: :patch,
+              local: true) do |form| %>
+
+              <%= form.govuk_text_field :percentage_home,
+                                        label: controller_t('percentage_home'),
+                                        hint: controller_t('hint.percentage_home'),
+                                        value: number_to_currency(model.percentage_home, unit: ''),
+                                        class: 'govuk-input govuk-input--width-5 input-percentage',
+                                        suffix: '%' %>
```

Comment:
I think the govuk-input is superflous here.

Reply:
Removed, and also removed from `app/views/citizens/other_assets/_second_home_conditional_checkbox.html.erb`.